scripts/install_astyle.sh

```diff
@@ -80,9 +80,12 @@ fi
     # Put everything in this directory
     FILESDIR=$INSTALL_ROOT/$ASTYLE_VER
 
-    svn checkout ${REPO_URL}/${ASTYLE_VER}/AStyle "$workdir"
-
+    git clone --sparse ${REPO_URL} "$workdir"
     cd "$workdir"
+    git sparse-checkout set AStyle
+    git checkout tags/${ASTYLE_VER}
+
+    cd "AStyle"
 
```

Comment:
Happy with this, but install_cppcheck.sh does it a different way (from https://stackoverflow.com/questions/791959) :-

```
git clone -b "$ASTYLE_VER" --depth 1 "$REPO_URL" "$workdir"
```

Reply:
It looks simpler! I like it.